io.github.junalmeida.webapps_manager.json

```diff
@@ -0,0 +1,55 @@
+{
+    "id": "io.github.junalmeida.webapps_manager",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.PySide.BaseApp",
+    "base-version": "6.7",
+    "command": "webapps_manager",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=host-os",
```

Comment:
why do you need this?

Reply:
The application purpose is to manage web applications and install an icon on the desktop to make it run like a native application, having its own icon on the taskbar. It does that using an installed Chrome-based browser or Firefox. 
To download the url title and favicon, it needs network. To list current installed browsers it needs access to host binaries to check if a supported browser binary lives in `/run/host/bin` .